nextcord/abc.py

```diff
@@ -117,8 +117,17 @@ class User(Snowflake, Protocol):
     ----------
     name: :class:`str`
         The user's username.
+    global_name: :class:`str`
+        The user's global name.
+
+        .. versionadded:: 2.6
     discriminator: :class:`str`
         The user's discriminator.
+
+        .. warning::
+          This field is deprecated, and will only return if the user has not yet migrated to the
+          new [username](https://dis.gd/usernames) update.
```

Comment:
Not too sure about the indentation here. This isn't markdown either ![link fail](https://cdn.discordapp.com/attachments/802586580766162967/1121233050878083142/image.png)